### DISCUSSION TITLE
Where is the playbook section in 2.4.70?

### DISCUSSION THREAD

**Discussion by Pencat89** [Security-Onion-Solutions/securityonion#13121] [Unsupported Versions]
Did this get removed in 2.4.70?

I'd like to disable a bunch of rules because I'd like to switch over to the new detections, but I can't find where the old playbooks are.

Can anyone assist?

**Comment by dougburks** [comment]
Have you looked in Detections to see if they were imported there?
https://docs.securityonion.net/en/2.4/detections.html
